deployment/ansible/roles/cac-tripplanner.app/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Install packages
+  apt: name={{ item }} state=present
+  with_items:
+    - postgresql-9.3
+    - postgresql-9.3-postgis-2.1
+    - libpq-dev
+    - python-dev
+    - python-psycopg2
+
+- name: Install pip packages
+  pip: name={{ item }}
+       virtualenv=/app_env
+  with_items:
```

Comment:
We just migrated HunchLab away from requirements.txt to keeping the requirements in Ansible like this; I think that's cleaner because it keeps the application dependencies and versions within Ansible. If you use requirements.txt then you have to ensure that the requirements.txt file exists on whatever machine you're running Ansible against.